feature-libs/order/components/order-history/order-history-extended/consolidated-information/order-consolidated-information.component.ts

```diff
@@ -0,0 +1,105 @@
+/*
+ * SPDX-FileCopyrightText: 2023 SAP Spartacus team <spartacus-team@sap.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import {
+  Component,
+  ChangeDetectionStrategy,
+  Input,
+  inject,
+} from '@angular/core';
+import { OrderEntry } from '@spartacus/cart/base/root';
+import { Images } from '@spartacus/core';
+import { OrderConsignmentsService } from '../../../order-details';
+import { Consignment, Order, OrderHistory } from '@spartacus/order/root';
+
+@Component({
+  selector: 'cx-order-consolidated-information',
+  templateUrl: './order-consolidated-information.component.html',
+  changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class OrderConsolidatedInformationComponent {
+  protected orderConsignmentsService = inject(OrderConsignmentsService);
+  @Input()
+  order?: OrderHistory;
+  private IMAGE_COUNT = 4; //showing fixed no.of images, without using carousel
```

Comment:
[QUESTION]
Why this property is `private`? Did you consider making it `protected`? What pros/cons of both solutions do you see?

Reply:
done